examples/azure-rules.yaml

```diff
@@ -656,4 +656,21 @@ groups:
           Please, check the naming convention in case the resource configuration is done correctly.
            
           More information at https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability#create-sap-hana-cluster-resources
+        scored: true
+  - id: 1.6
+    description: "Check OS and package versions are supported and up-to-date"
+    checks:
+      - id: 1.6.1
+        description: "Check os release"
+        audit: 'cat /etc/os-release'
+        tests:
+          bin_op: or
+          test_items:
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 12 SP4"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15 SP1"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15 SP2"'
+            - flag: 'PRETTY_NAME="SUSE Linux Enterprise Server for SAP Applications 15 SP3"'
+        remediation: |
+          Install a proper OS release
```

Comment:
"proper" is maybe a bit harsh. how about "SUSE Supported OS release" ?

Reply:
I think this check will fail. SLES for SAP and SLES have the same content on os-release for historical reasons and certification matters. Both state SLES on it. if we need to check the commercial product, the /etc/products.d/baseproduct file should be used instead.

Also, SLES is a certified OS for SAP, so the check should consider it too.

Reply:
Thaks! I changed "proper" -> "SUSE supported". The /etc/os-release indeed should never be used for examining the base product (https://www.suse.com/support/kb/doc/?id=000019341). I check it no by the /etc/products.d/baseproduct

Reply:
Thanks!